### DISCUSSION TITLE
Upgrade SmartFormat package to v3.0.0

### DISCUSSION THREAD

**Discussion by axunonb** [gitextensions/gitextensions#9909] [General]
Hi,
You're using the [`SmartFormat` package](https://github.com/axuno/SmartFormat) for translations.
We bumped it to v3.0.0, while you're using v2.7.0.
I forked your repo and gave the upgrade to v3 a try with almost no effort.
It should even work with the core package, having less dependencies than the full package
=> Would you like me to create PR to the master branch?
Maybe have a look at the [Why Migrate](https://github.com/axuno/SmartFormat/wiki/Why-Migrate) Wiki Chapter to see that it would pay back in performance and memory footprint.

**Comment by gerhardol** [comment]
> Would you like me to create PR to the master branch?

Yes please

**Reply by axunonb** [reply]
Done, PR https://github.com/gitextensions/gitextensions/pull/9912
